include/arcade/IGame.hpp

```diff
@@ -20,6 +20,7 @@
 namespace arcade
 {
     struct Event;
+    struct vec2u;
```

Comment:
Does that compile? vec2u is not a structure, but a type alias to `vec2<unsigned int>`. Moreover forward declarations only work when you consume the types behind pointers or references, that is not the case of the loadAssets method.